README.md

```diff
@@ -1638,6 +1639,24 @@ print(schema_graph.get_unique_indexes_for_class('Animal'))
 
 For more information about the `SchemaGraph`, please see the class documentation. 
 
+
+### Cypher query parameters
+Redisgraph doesn't support query parameters, so we perform manual parameter interpolation in the
+graphql_to_redisgraph_cypher function. However, for Neo4j, we can use Neo4j's client to do 
```

Comment:
micro-nit: `graphql_to_redisgraph_cypher` (this way it appears highlighted as code in markdown)

Reply:
^ What I think @pmantica1 is referring to is that you should use backticks ("`" characters) to wrap things that should appear in fixed-width font. This is the difference between graphql_to_redisgraph_cypher and `graphql_to_redisgraph_cypher`.